lib/collection/src/operations/universal_query/planned_query.rs

```diff
@@ -131,8 +162,8 @@ fn recurse_prefetches(
                         params,
                         limit,
                         offset: 0,
-                        with_payload: None,
-                        with_vector: None,
+                        with_payload: Some(WithPayloadInterface::Bool(false)),
```

Comment:
`None` is not good enough?

Reply:
I figured it would be better to make it explicit :)